Makefile.dep

```diff
@@ -1,3 +1,7 @@
+ifneq (,$(filter log_printfnoformat,$(USEMODULE)))
+	DISABLE_MODULE += log_printf
```

Comment:
(in conjunction with https://github.com/RIOT-OS/RIOT/pull/2503/files#diff-9cfca3af95b153db2dacedca47e308f1R4)